lib/ssh_scan/database.rb

```diff
@@ -0,0 +1,30 @@
+# API Databases
+require 'ssh_scan/database/mongo'
+require 'ssh_scan/database/sqlite'
+
+module SSHScan
+  class DatabaseConfig
+    def initialize(opts = {})
+      @@db_type = if !opts || opts.empty?
```

Comment:
On using `@db_type` instance variable here. In the `set_database` class method, `@db_type` is empty.